README.md

```diff
@@ -62,9 +62,16 @@ Supported keys of the map are:
 - `:category` - a `String`
 - `:data` - a map with `String` keys and `String` values
 
+### Message
+
+When an event has a `:message` key, the following data should be contained within a map, thus:
+
+- `:message` - A `String`
```

Comment:
```suggestion
- `:formatted` - A `String` with the interpolated message.
- `:message` - A `String`
```

Reply:
Hi,

I've just done a bit of testing and it doesn't seem to follow the documentation.

In all examples, params was set to `dharrigan`

If I `setFormatted` as `User %s is logged in`, then on Sentry it shows `User %s is logged in`
If I `setMessage` as `User %s is logged in`, then on Sentry it shows `User dharrigan is logged in`

If looks like `setFormatted` is the raw string, whereas `setMessage` is the string to be interpolated.

Reply:
Both names are far from ideal so I understand the confusion